src/subscribe/index.html

```diff
@@ -73,6 +73,10 @@
                 </section>
                 <section class="form-row">
                     <h5>Organization</h5>
+                    <p>
+                        You will start a sixty-day free trial when you complete signup. We collect your credit card
+                        now so that you can seamlessly continue using Stockpile when your trial runs out.
+                    </p>
```

Comment:
[Apparently not](https://stripe.com/docs/subscriptions/trials):
> When creating a subscription with a trial period, no payment method is required for the customer. An immediate invoice is still created, but for $0.

For now, though, I think we should require it, mainly because allowing the user to subscribe without payment information and then add it later will require a significant amount of development time. I created #29 for adding this eventually.

But also, requiring a credit card filters out potential customers who are unlikely to actually ever buy a subscription, so this could work in our favor for the time being.

I will change the description to include "email info@stockpileapp.co at any time to cancel your trial." I do like the idea of notifying the customer before their trial ends, but we don't have email support on the API yet, and we would need to implement the Stripe webhook in order to receive the "trial expires in three days" event sent by Stripe.